src/Entity/Media.php

```diff
@@ -99,7 +99,7 @@ public function createdAt(): DateTimeType
         return $this->createdAt;
     }
 
-    public function updatedAt(): DateTimeType
+    public function updatedAt(): ?DateTimeType
```

Comment:
After a brief discussion: `updatedAt` is not nullable throughout the framework. Do not make it nullable here nor in the `Definition` below.